_includes/getting-started.html

```diff
@@ -109,126 +120,118 @@ <h4 style="font-size: 1.5em;">Using Helm</h4>
+              <div id="start" class="highlight-code highlight-source-shell" style="margin-left:1.8em;">
+                <div style="width:auto; height:30px; padding:0px; position:relative;">
+                  <pre><code>   $ ./mesheryctl start
               </code></pre>
-              <!-- copy to clipboard -->
-              <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
-                style="position: absolute;top:-14px; right: -1px; border: 0; 
+                  <!-- copy to clipboard -->
+                  <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4" style="position: absolute;top:-14px; right: -1px; border: 0; 
                     cursor: pointer; background:#00a4ca; width:60px; height:44px; margin-bottom:35px; 
-                    border-radius: 2px; margin:0px 0px 880px 0px;"
-                data-clipboard-target="#start"
-                data-clipboard-text="./mesheryctl start">
-                <img src="/images/buttons/clippy.svg" style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
-              </a>
+                    border-radius: 2px; margin:0px 0px 880px 0px;" data-clipboard-target="#start"
+                    data-clipboard-text="./mesheryctl start">
+                    <img src="/images/buttons/clippy.svg"
+                      style="width:20px; margin: 0; position: absolute; top: 50%; left: 54%; transform: translate(-50%, -50%);" />
+                  </a>
+                </div>
+              </div>
+
+            </div>
           </div>
         </div>
-          
-        </div>  
-  </div>
-</div>
-<hr>
-    <div class="login">
-      <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
-    <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
-    <p style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.</p>
-    </div>
-    
-    <!-- <h3 style="color:#111111;">Step 3: Configure and Enjoy Meshery</h3>
-    <p>- Configure Meshery with references to your deployed adapters, using "meshery-&lt;service-mesh-name&gt;:&lt;port&gt;" (e.g. <code>meshery-istio:10000</code> or <code>meshery-linkerd:10001</code>)
-      in the <i>Mesh Adapter Location</i> field.</p> -->
+        <hr>
+        <div class="login">
+          <h2 style="margin-top: 1em;"><strong>Login</strong></h2>
+          <h1 style="font-size: 80px;opacity: 0.2;margin-top: -95px;margin-left: -55px;"><strong>02</strong></h1>
+          <p id="breakWords" style="margin-left:1em;">Access Meshery in your browser if you are using Kubernetes at <a
+              href="http://127.0.0.1:9081" style="color: rgb(85, 217, 240);">http://127.0.0.1:9081</a>.</p>
+        </div>
+      </div>
```

Comment:
@gunjanmimo There seems to be margin and padding issue over here. To fix margin and padding in parent elements also needs to be fixed

Desktop View 
Text in login section  is overflowing in x-direction
![Screenshot from 2020-06-16 11-45-22](https://user-images.githubusercontent.com/35156299/84741417-b8646d80-afcc-11ea-94d7-c7628117426e.png)